site/gatsby-site/server/MongoDBAdapter.ts

```diff
@@ -0,0 +1,251 @@
+/**
```

Comment:
instead of importing this from the npm package I copy pasted the file here because it is a ESM only package, and it wasn't playing nice with Gatsby babel config. (Not worth getting it to work for a package with only one file)